src/renderer/src/components/interaction/NewInteractionTemplate.tsx

```
@@ -106,52 +114,70 @@ const NewInteractionTemplate = (): ReactElement => {
+        <FormLabel>{t('Participant Instructions On Complete')}</FormLabel>
         <Textarea
           value={participantInstructionsOnComplete}
           onChange={handleChangeParticipantInstructionsOnComplete}
         />
         <FormHelperText>
-          These are the instructions that are shown to the participant when the
-          interaction has been marked as completed.
+          {t(
+            'These are the instructions that are shown to the participant when the interaction has been marked as completed.',
+          )}
+          '
```

ESLint

[error] `'` can be escaped with `&apos;`, `&lsquo;`, `&#39;`, `&rsquo;`.

(react/no-unescaped-entities)